MAUI/TreeMap/migration.md

```diff
@@ -0,0 +1,1456 @@
+---
+layout: post
+title: Migrate from Xamarin to .NET MAUI TreeMap (SfTreeMap) | Syncfusion
```

Comment:
Migrate from Xamarin.Forms to .NET MAUI TreeMap (SfTreeMap) | Syncfusion

Reply:
Changed title